src/examples/colors/generate_wcs_languages.py

```diff
@@ -75,25 +77,22 @@ def generate_color_languages(num_languages = -1,
     language_name_from_code = dict()
     language_terms = dict()
 
-
-
-    #Get current dir for relative paths
+    # Get current dir for relative paths
     current_dir = os.path.dirname(os.path.realpath(__file__))
 
-    #Get the expression from a language based on the color name
-    def get_expression_from_language(color_name:str, language:ColorLanguage):
+    # Get the expression from a language based on the color name
+    def get_expression_from_language(color_name: str, language: ColorLanguage):
         for expression in language.expressions:
             for referent in expression.meaning.referents:
                 if referent.name == color_name:
                     return expression
         return None
 
-
     # if USE_NOGA_ARRAYS:
     #     model_data = load_noga_model(filename=f"{current_dir}/model/model.pkl", model_dir=f"{current_dir}/model/")
 
-    #Generate all WCS color codes
-    #Convert the Munsell hues of the WCS data to CIELab data 
+    # Generate all WCS color codes
+    # Convert the Munsell hues of the WCS data to CIELab data
     munsell_to_cielab = {}
     referents_by_color_code = {}
     with open(
```

Comment:
make this filename an arg to the method, not hard-coded

Reply:
Better yet: I think this generating of the universe can be done once as a separate step and then saved and re-loaded here